google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryOptions.java

```diff
@@ -81,6 +93,19 @@ protected BigQueryRpcFactory getDefaultRpcFactory() {
   }
 
   @Override
+  public TransportOptions getDefaultTransportOptions() {
+    return getDefaultHttpTransportOptions();
+  }
+
+  public static HttpTransportOptions getDefaultHttpTransportOptions() {
+    return HttpTransportOptions.newBuilder().build();
+  }
+
+  public HttpTransportOptions getHttpTransportOptions() {
```

Comment:
Hmm @michaelbausor you make a good point. I was trying to clean up the Default*Rpc classes with this change, but it encourages bad behavior to users.